validation.js

```diff
@@ -0,0 +1,26 @@
+const $form = document.querySelector('.contactform');
+const $mail = document.querySelector('.email');
+const $message = document.querySelector('.warnings');
+const $closemenu = document.querySelector('.x-warning');
+const $container = document.querySelector('.hide');
+
+$form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const restriction = /[A-Z]/;
+  let warnings = '';
+  let enter = false;
+
+  $closemenu.addEventListener('click', () => {
+    $container.classList.add('hide');
+  });
+
+  if (restriction.test($mail.value)) {
+    warnings += 'Warning: Please use only lower case for email';
```

Comment:
- [ ] As seen in the image below showing the project requirement your error message is supposed to be displayed close to the submit button or input field not as a popup like you have done. I kindly suggest you make this change to your project so that it is in line with the project requirement 👍 ✨ 
- project requirement
![pr](https://github.com/eamador1/portfolio-project/assets/89055502/dc05addc-9118-4744-a6d6-3492118c8ce4)
- your design
![yr](https://github.com/eamador1/portfolio-project/assets/89055502/0ea7905a-10bb-473c-b5e7-8e70ee8c9b6b)